src/common/perf_counters.cc

```diff
@@ -354,9 +374,27 @@ void PerfCounters::reset()
 }
 
 void PerfCounters::dump_formatted_generic(Formatter *f, bool schema,
-    bool histograms, const std::string &counter) const
+    bool histograms, bool dump_labeled, const std::string &counter) const
 {
-  f->open_object_section(m_name.c_str());
+
+  if(dump_labeled) {
+    std::string_view perf_counter_name = ceph::perf_counters::key_name(m_name);
+    f->open_object_section(perf_counter_name);
+
+    f->open_object_section("labels");
```

Comment:
This would definitely require a new option/command though.  With something like `__labels__`, since it's basically just one distinctly named key in an existing dictionary, we could still potentially explore just dumping the new stuff unconditionally.

Reply:
> we could still potentially explore just dumping the new stuff unconditionally with a release note.

I think this is worth bringing up in tomorrow's CDM meeting.